ibis/backends/pandas/client.py

```diff
@@ -86,9 +86,9 @@
     'integer': dt.int64,
     'mixed-integer': dt.binary,
     'mixed-integer-float': dt.float64,
-    'decimal': dt.binary,
+    'decimal': dt.float64,
     'complex': dt.binary,
-    'categorical': dt.binary,
+    'categorical': dt.category,
```

Comment:
are there paths for this at all in the tests?

Reply:
yep! exercised here:
https://github.com/ibis-project/ibis/pull/2792/files#diff-b3fc0a3096849288f88fe283547dd9ac56c8590b7bd36ffdcf88084dda172c0aL182